app/notebook-store.js

```diff
@@ -92,6 +93,11 @@ function _getFilename (req) {
     return name;
 }
 
+function _hiddenFileFilter (filename) {
+    //check for hidden files or directories, returns true if file is not hidden
+    return !(/(^|\/)\.[^\/\.]/g).test(filename);
```

Comment:
Yes, we could since we can assume readdir will return the content only.  The above might be more complete, but not not needed.  I'll update the PR.